force_wfmanager/left_side_pane/icons/license.txt

```diff
@@ -0,0 +1,14 @@
+The icons are mostly derived work from other icons. As such they are
+licensed accordingly to the original license:
+
+Project                 License                   File
+----------------------------------------------------------------------------
+Enthought               BSD 3-Clause              LICENSE.txt
+
+Unless stated in this file, icons are the work of Enthought, and are
+released under a 3 clause BSD license.
+
+Files and orginal authors:
+----------------------------------------------------------------------------
+force-wfmanager/force_wfmanager/left_side_pane/icons:
+                            all | Enthought
```

Comment:
I am unsure if the size of the invalid icon is too big compared to the available space.